Application/EdFi.Admin.DataAccess/Extensions/ModelBuilderExtensions.cs

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace EdFi.Admin.DataAccess.Extensions;
+
+public static class ModelBuilderExtensions
+{
+    public static void MakeDbObjectNamesLowercase(this ModelBuilder modelBuilder)

```

Comment:
@simpat-jesus , I realized it would have been better to implement this class's functionality via the IConvention interfaces in EF Core. I can make that change if needed

Reply:
We may leave this for another ticket;I'm still not sure which interface implementing IConvention should be used